_posts/2016-06-01-welcome-to-magenta.markdown

```diff
@@ -29,7 +29,7 @@ Magenta, we want to explore the other side—developing algorithms that
 can learn how to generate art and music, potentially creating
 compelling and artistic content on their own.
 
-Second, Magenta is an attempt to build a community of artists, coders
+Second, Magenta is an attempt to build a community of artists, coders,
```

Comment:
I prefer leaving this as is (no Oxford comma). This might be a U.S. / U.K. difference in preference. There is a great Vampire Weekend song about the debate, "Oxford Comma" :-)